src/hammer-vlsi/defaults.yml

```
@@ -842,6 +874,44 @@ sim.submit:
   settings: "vlsi.submit.settings"
   settings_meta: lazycrossref
 
+# Specific inputs for the power tool.
+# These may vary based on the specific tool plugin
+power.inputs:
+  # The path to a place and route database of the design to be analyzed
+  # type: str
+  database: null
+
+  # Enable static power rail analysis
+  # type: bool
+  static_rail: false
+
+  # The name of the test bench from the simulation.
+  # type: str
+  tb_name: "TestDriver"
+
+  # The name of the device under test (DUT) from the simulation.
+  # type: str
+  tb_dut: "dut"
+
+  # The paths of all spef (extraction) files needed for the design.
+  # This may include a file per MMMC corner
+  # type: List[str]
+  spefs: null
```

Should this be an empty list?
```suggestion
  spefs: []
```

---

Fixed.